### PR TITLE
[TASK] Add some packagist badges

### PR DESCRIPTION
**Pull request by egobude**

### CODE REVIEW

**Comment by robertlemke** [comment]
Sure, why not – but I think the number of downloads is a bit confusing, because this is about the development collection and not about Neos. People might not get that.

File: Readme.rst [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
    :target: https://codeclimate.com/github/neos/neos-development-collection
-   :alt: Code Climate
+.. |Travis Build Status| image:: https://travis-ci.org/neos/neos-development-collection.svg
+   :target: https://travis-ci.org/neos/neos-development-collection
+.. |Latest Stable Version| image:: https://poser.pugx.org/neos/neos-development-collection/v/stable
+   :target: https://packagist.org/packages/neos/neos-development-collection
+.. |Total Downloads| image:: https://poser.pugx.org/neos/neos-development-collection/downloads
+   :target: https://packagist.org/packages/neos/neos-development-collection
+.. |Latest Unstable Version| image:: https://poser.pugx.org/neos/neos-development-collection/v/unstable
+   :target: https://packagist.org/packages/neos/neos-development-collection
```
Comment (aertmann) [inline Readme.rst]: I'd skip this one since it doesn't really add any value as I see it.

**Comment by aertmann** [comment]
I agree with Robert here, that download count can be confusing and doesn't add much value.

However it does make sense for some of the other repositories.

File: Readme.rst [hunk 2]
```diff
@@ -1,8 +1,18 @@
-.. image:: https://travis-ci.org/neos/neos-development-collection.svg
-    :target: https://travis-ci.org/neos/neos-development-collection
-.. image:: https://codeclimate.com/github/neos/neos-development-collection/badges/gpa.svg
+|Code Climate| |Travis Build Status| |Latest Stable Version| |Total Downloads| |Latest Unstable Version|
```
Comment (hlubek) [inline Readme.rst]: Can we have the Travis build status as the first item? IMHO it's one of the most important ones.

**Comment by egobude** [comment]
Thanks for the comments. I will change that :)

**Comment by egobude** [comment]
@see: https://github.com/neos/neos-development-collection/pull/93

**Pull request closed without merging**